testsuite/tests/parsetree/source.ml

```diff
@@ -5052,10 +5052,10 @@ let get_buf s i =
 let set_buf s i u =
   let n = UChar.uint_code u in
   begin
-    s.[i] <- Char.chr (n lsr 24);
-    s.[i + 1] <- Char.chr (n lsr 16 lor 0xff);
-    s.[i + 2] <- Char.chr (n lsr 8 lor 0xff);
-    s.[i + 3] <- Char.chr (n lor 0xff);
+    s.![i] <- Char.chr (n lsr 24);
```

Comment:
I’m a bit surprised by this change. Where is the `(.![]<-)` operator defined? I wasn’t able to locate it

Reply:
This test only checks that the source file can be parsed.

Reply:
ah I see. Thanks!